### PR TITLE
Dev 36 - implemented user avatars at the Register page

### PR DESCRIPTION
**Pull request by KovDimaY**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure all the tests are still passing.
* [x] Make sure the test coverage is still 100%.
* [x] Make sure your source branch is correct.
* [x] Make sure your destination branch is **devel**.
* [x] Make sure the name of the PR has a correct DEV-# identification prefix.
* [x] Make sure you reviewed all your changes and they look correct.
* [x] Make sure all the commits have a correct format. 
* [x] Make sure you refer related issues for you PR. 
* [x] Make sure you add **KovDimaY** as a reviewer and assignee of your PR.

Take a look at the Issue #63

### CODE REVIEW

File: client/src/components/User/index.js [hunk 1]
```diff
@@ -13,28 +13,19 @@ const User = (props) => {
...
-          {user.name}
+          <span>Name:</span>{' '}{user.name}
         </div>
         <div>
-          <span>
-            Lastname:
-          </span>
-          {' '}
-          {user.lastname}
+          <span>Lastname:</span>{' '}{user.lastname}
```
Comment (NikolayYakovenko) [inline client/src/components/User/index.js]: If you really need space here you can use `&nbsp;` symbol:
```
<div>
    Lastname:&nbsp;{user.lastname}
</div>
```

File: client/src/components/User/__tests__/__snapshots__/index.spec.js.snap [hunk 2]
```diff
@@ -5,15 +5,20 @@ exports[`<User /> should render component 1`] = `
...
   >
     <div
-      className="avatar-picture"
+      className="user-avatar-container fit-parent"
     >
-      <img
-        alt="avatar"
-        src="/images/avatar.png"
-      />
+      <div
```
Comment (KovDimaY) [inline client/src/components/User/__tests__/__snapshots__/index.spec.js.snap]: Avatar should be mocked. Changes in avatar should not affect snapshot of the User component

File: client/src/components/User/index.js [hunk 3]
```diff
@@ -13,28 +13,19 @@ const User = (props) => {
...
+      </div>
+
       <div className="data">
         <div>
-          <span>
-            Name:
-          </span>
-          {' '}
-          {user.name}
+          <span>Name:</span>{' '}{user.name}
```
Comment (KovDimaY) [inline client/src/components/User/index.js]: I am not sure if it is really needed (the space in brackets), but ESlint hardly "suggested" me to do it this way

File: client/src/components/UserAvatar/styles.css [hunk 4]
```diff
@@ -1,19 +1,16 @@
...
-.user-avatar-container .avatar-picture {
-    text-align: center;
-    margin-top: 30px;
-    margin-bottom: 20px;
-}
-
 .user-avatar-container .avatar-picture img {
-    width: 100%;
     max-width: 200px;
+    max-height: 200px;
```
Comment (NikolayYakovenko) [inline client/src/components/UserAvatar/styles.css]: Check this example https://codepen.io/anon/pen/PydRLE
Try add **max-width/width** to image wrapper.
It is not mistake, but can help to manage layout on small screens. You just change properties for only one element.

File: client/src/containers/Register/styles.css [hunk 5]
```diff
@@ -34,35 +30,77 @@
...
+        height: 50px;
+    }
+    .register-container .user-names {
+        width: calc(100% - 75px);
+        margin: 3px 0 0 25px;
+        text-align: left;
+    }
+    .register-container .user-names p {
+        margin: 0;
+        text-overflow: ellipsis;
```
Comment (KovDimaY) [inline client/src/containers/Register/styles.css]: Here it maybe can be deleted as it present in the general styles

File: client/src/containers/EditUserProfile/styles.css [hunk 6]
```diff
@@ -1,65 +1,58 @@
+.edit-user-profile-container {
+    max-width: 600px;
+    margin: 0 auto 75px auto;
+}
 .edit-user-profile-container .avatar {
     text-align: center;
     margin-top: 30px;
 }
-
 .edit-user-profile-container .avatar img {
```
Comment (NikolayYakovenko) [inline client/src/containers/EditUserProfile/styles.css]: You should not change inner styles of **UserAvatar** component in the other component.
Default behavior of UserAvatar is render image. This component take all available place (with `max-widht` property)
If you want to change width use props to set width for image.
Or, what I prefer more, wrap **UserAvatar** into div with class, which sets width.

Comment (KovDimaY) [reply client/src/containers/EditUserProfile/styles.css]: Great point! Thank you! I missed this selector! 👍

**Pull request merged**